subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -172,7 +186,7 @@ def _generate_pod_timings(pod_times, start, end):
 
 def _gather_prom_metrics(entry) -> dict:
     output = {}
-
+    prom.register()
```

Comment:
🤔  I think I removed this call in a previous commit, because it was triggering multiple `register` 🤔 

https://github.com/openshift-psap/ci-artifacts/commit/7fc95b2248842b5bbde603988c86bc5b313faa2f

my bad, I guess it was "unnecessary" after all 🙃 

Could you tell me which command doesn't work when it's missing? we need to find a location where it's called only once.

The "main" place from where it's called is:
https://github.com/openshift-psap/ci-artifacts/blob/main/subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/plotting/__init__.py#LL23C1-L23C20

Reply:
The problem is when parsing to a  LTS payload.  I use the prom report to start registering metrics, so if it's not included here, no metrics are a part of the payload.

Reply:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/721/pull-ci-openshift-psap-ci-artifacts-main-ods-notebooks-light/1661820915014438912/artifacts/notebooks-light/test/artifacts/002__rhods_notebook_performance/015__plots/_matbench_generate_lts.log

here is the trace related to this being called twice
I wonder why that's happening only with notebook performance visualization 🤔